mynearwallet/my-near-wallet
# chore: add storybook

## Description
* Added storybook

## Work Done
* Configured and installed storybook
* Converted some components to `tsx`
* Changed old translations to use new translations for component
* Added `typescript` and `webpack` in dev dependencies as this is required for storybook to work

## How to test
* Run `yarn storybook` in frontend folder
* You should see storybook open with a single component 

## Screenshot
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/46312751/232679222-25d53efc-aea1-4afd-acc0-d3c4822d80fd.png">
